app/src/main/java/com/example/bigowlapp/activity/SearchContactsToSupervise.java

```diff
@@ -88,10 +88,23 @@ private void loadContacts() {
         listContactsView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             // argument position gives the index of item which is clicked
             public void onItemClick(AdapterView<?> arg0, View v, int position, long arg3) {
-                String number = (String) listContactsView.getItemAtPosition(position);
+                //String number = (String) listContactsView.getItemAtPosition(position);
+                String contactDetails = (String) listContactsView.getItemAtPosition(position);
+                String contactNumber;
+                if (contactDetails.split("\\n").length < 2) {
+                    contactNumber = contactDetails
+                            .split("\\n")[0]
+                            .replaceAll("[^+0-9]", "");
+                } else {
+                    contactNumber = contactDetails
+                            .split("\\n")[1]
+                            .replaceAll("[^+0-9]", "");
+                }
+
 
                 db.collection("users")
-                        .whereEqualTo("phoneNumber", number.replaceAll("[^+0-9]", ""))
+                        //.whereEqualTo("phoneNumber", number.replaceAll("[^+0-9]", ""))
```

Comment:
Same thing as above; I think you can remove this comment too.

Reply:
ok. It has been removed